man/man8/zpool.8

```diff
@@ -2235,6 +2241,19 @@ Specify \fBu\fR for a printed representation of the internal representation of t
 .sp
 .ne 2
 .na
+\fB\fBzpool sync\fR\fR [\fBpool\fR] ...
+.ad
+.sp .6
+.RS 4n
+This command will trigger TXG sync(s) and will not return until the data is
+committed to the pool(s).
```

Comment:
Let's wordsmith this a bit to avoid referring to a txg which in an internal detail, instead let's describe the expected behavior.  The two critical things worth mentioning are:

* Forces all in-core dirty data to be written to the primary pool storage and not the log device.
* Forces an update of administrative information including quota reporting.